deploy/contents/install/app/install-utils.sh

```diff
@@ -562,6 +562,41 @@ function array_contains_or_empty () {
     return 1
 }
 
+function count_services {
+    parse_env_option "$1"
+    services_count=$((services_count+1))
+    if [ -n "${CP_SERVICES_ENABLED}" ]; then 
+        export CP_SERVICES_ENABLED="${CP_SERVICES_ENABLED},${1}"
+    else 
+        export CP_SERVICES_ENABLED="${1}"
+    fi
+    }
+
+function export_service_all {
+        export CP_INSTALL_SERVICES_ALL=1
+        export CP_SERVICES_ENABLED="all"
+    }
+
+function services_from_pitc {
```

Comment:
it is better if name of the function is an action (because function is actually a some action)
f.i. this one will `enable` services for the installation from point-in-time configuration, right? so let's name it appropriate

Reply:
Also it is better to not to use abbreviations (`pitc`) in the name if the function if such abbreviation is not `well-known`